src/main/java/zd/zero/waifu/motivator/plugin/integrations/WaifuConsoleFilterProvider.kt

```diff
@@ -0,0 +1,52 @@
+package zd.zero.waifu.motivator.plugin.integrations
+
+import com.intellij.execution.filters.ConsoleFilterProvider
+import com.intellij.execution.filters.Filter
+import com.intellij.openapi.project.Project
+import zd.zero.waifu.motivator.plugin.alert.AlertConfigurationAllEnabled
+import zd.zero.waifu.motivator.plugin.assets.WaifuAssetCategory
+import zd.zero.waifu.motivator.plugin.motivation.MotivationFactory
+import zd.zero.waifu.motivator.plugin.motivation.event.MotivationEvent
+import zd.zero.waifu.motivator.plugin.motivation.event.MotivationEventCategory
+import zd.zero.waifu.motivator.plugin.motivation.event.MotivationEvents
+import zd.zero.waifu.motivator.plugin.settings.WaifuMotivatorPluginState.getPluginState
+
+class WaifuConsoleFilterProvider : ConsoleFilterProvider {
+
+    private var prevLength: Int = Int.MAX_VALUE
+
+    private var alertShown = false
+
+    override fun getDefaultFilters(project: Project): Array<Filter> = when {
+        getPluginState().isLogWatcherEnabled -> arrayOf(
```

Comment:
I made this to help out https://github.com/Unthrottled/demo-spring

Reply:
![Peek 2020-10-31 09-44](https://user-images.githubusercontent.com/15972415/97782106-a4756e80-1b5d-11eb-989b-2d18a2b7da90.gif)

I've got a temporary workaround, it will only do it once if I run it through the verify. But it might get annoying if I am debugging a test.